python/whylogs/api/writer/local.py

```diff
@@ -1,30 +1,36 @@
+import logging
 import os
-from typing import Optional
+from typing import Optional, Union
 
 from whylogs.api.writer import Writer
+from whylogs.api.writer.writer import Writable
 from whylogs.core import DatasetProfileView
+from whylogs.core.utils import deprecated_alias
+
+logger = logging.getLogger(__name__)
 
 
 class LocalWriter(Writer):
     def __init__(self, base_dir: Optional[str] = None, base_name: Optional[str] = None) -> None:
         if base_dir is None:
             base_dir = os.getcwd()
-        if base_name is None:
-            base_name = "profile"
-
         self._base_dir = base_dir
         self._base_name = base_name
 
-    def write(self, profile: DatasetProfileView, dest: Optional[str] = None) -> None:
-        if dest is None:
-            dest = f"{self._base_name}_{profile.creation_timestamp}.bin"
+    @deprecated_alias(profile="file")
+    def write(
+        self,
+        file: Optional[Union[Writable, DatasetProfileView]] = None,
```

Comment:
Perhaps just `file: Optional[Writable] = None`, though this will crash if the file is `None`?

Reply:
Yeah, looking at `Writer::write()` this should just be `file: Writable`